youtube_dl/extractor/sbs.py

```diff
@@ -44,12 +44,13 @@ class SBSIE(InfoExtractor):
             (["\'])(?P<url>https?://(?:www\.)?sbs\.com\.au/ondemand/video/.+?)\1''']
 
     _TESTS = [{
-        # Original URL is handled by the generic IE which finds the iframe:
+        # Exceptional unrestricted show for testing, thanks SBS,
```

Comment:
What is this comment supposed to mean?

Reply:
The test URL was found in an `<iframe>` of a page that no longer exists.